sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosEndToEndOperationLatencyPolicyConfig.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.CosmosEndToEndOperationLatencyPolicyConfigBuilder;
+
+import java.time.Duration;
+
+/**
+ * Represents End to end operation latency policy config
+ * This enables requests to get cancelled by the client once the specified timeout is reached
+ */
+public class CosmosEndToEndOperationLatencyPolicyConfig {
+    /**
+     * The default end to end operation latency policy with timeout of 2 seconds
+     */
+    public static final CosmosEndToEndOperationLatencyPolicyConfig DEFAULT =
+        new CosmosEndToEndOperationLatencyPolicyConfigBuilder().build();
```

Comment:
hmm, generally I do not see we reference builder class inside the model class

Reply:
Removed